### PR TITLE
Fix chat input text color (Fixes #4304)

### LINKED ISSUE #4304: Improve the chat improvement/coloring
This is to tweak some improvements from #3946 
While typing a new message, the text I'm typing will turn "disconnected" red, or "connected" green.
Interestingly enough, if you keep the message unsent long enough (while typing), it will sometimes turn normal white again.

Also, a suggestion, maybe have the ability to set colors to the usernames in the chat, based on group names? (I can throw this into a new issue request as well if you'd like)

**Screenshots / Video:**
image

### PR DESCRIPTION
**Pull request by wolfreak99**
When there was a red or green chat message at the top of the chat history, the chat input text would appear that color for some reason. I did some experimenting and using the console to test numbers, i found that 34 keeps the color white and outlined (including during storms/night cycle).. Matching up various definitions, COLOUR_FLAG_OUTLINED gives 32 and COLOUR_WHITE gives 2, so put them together and 34 is no longer a magic number..

TLDR: This keeps chat input text color a steady white outlined color.

### CODE REVIEW

**Comment by IntelOrca** [comment]
Almost right, but 42580af498fdebc71dde372de4368a9938569dd4 was the correct fix.
